src/helpers.cpp

```diff
@@ -16,7 +16,7 @@
 #define stat64 __stat64
 #endif
 
-#if defined(__APPLE__)
+#if defined(__APPLE__) || !defined (__GLIBC__)
```

Comment:
I don't think that change is correct - it would conflict with the define for `stat64` when `_MSC_VER` is defined (above)

Maybe (assuming that `__GLIBC__` isn't defined for `__APPLE__` anyway):

```
#ifdef _MSC_VER
#define stat64 __stat64
#elif !defined(__GLIBC__)
#define stat64 stat
#endif
```